frontend/src/utils/hotkey_listener.ts

```diff
@@ -288,23 +288,22 @@ export class HotkeyListener {
     return binding.split(',').map(group => group.split('+').map(key => key.trim()));
   }
 
-  private checkSpecialKeyBindingPressed(bindings: string[][]): boolean {
+  private checkSpecialKeyBindingPressed(bindings: string[][], event: KeyboardEvent): boolean {
```

Comment:
Updated onSpecialKeydown/onSpecialKeyup logic because it had a bug where keyup hotkey handler would only fire if it had similar key down hotkey handler registered.